CMSgov/design-system
# [WNMGDS-2825] Fix release tagging

## Summary

WNMGDS-2825

- The release script no longer uses Lerna. Instead it manually creates the publish commit and tags.
- The release script no longer asks for version changes for individual packages. Instead, you just select the "release type" from a list, and it applies the change to all packages appropriately.
- Fixed the `--undo` sub-command because we only try to delete tags that we created (because we're in charge of that now).

## How to test

1. Check out the branch
2. Create your own test branch off of it like `git branch -b "your-test-branch-name"` so it doesn't push changes to origin and cause the Jenkins to do extra work with CI tests
3. Run the release script. Check the changes it made for correctness
4. Undo the release with `yarn release --undo`
5. Delete your test branch

## Checklist
